File: tech-studio-projects/financial-industry/algo-trading-platform/java/algo-trading-platform/src/com/fintech/algotrading/logging/LoggingConfiguration.java
package com.fintech.algotrading.logging;

public class LoggingConfiguration {

    private TextLogger textLoggerConfiguration;

    // Getter for TextLoggerConfiguration
    public TextLogger getTextLoggerConfiguration() {
        return textLoggerConfiguration;
    }

    // Setter for TextLoggerConfiguration
    public void setTextLoggerConfiguration(TextLogger config) {
        this.textLoggerConfiguration = config;
    }
}

File: tech-studio-projects/financial-industry/algo-trading-platform/cpp/logging/LoggingConfiguration.cpp
#include "LoggingConfiguration.h"

// Getter for TextLoggerConfiguration
std::shared_ptr<TextLoggerConfiguration> LoggingConfiguration::GetTextLoggerConfiguration() const {
    return textLoggerConfiguration;
}

// Setter for TextLoggerConfiguration
void LoggingConfiguration::SetTextLoggerConfiguration(const std::shared_ptr<TextLoggerConfiguration>& config) {
    textLoggerConfiguration = config;
}
